fastlane/fastlane
# [ci] Update CircleCI to start using Xcode14 for unit-tests

### Checklist

### Motivation and Context



### Description



### Testing Steps


Please check the CI run, all unit tests should pass.

<img width="775" alt="Screenshot 2022-09-10 at 17 16 16" src="https://user-images.githubusercontent.com/5364500/189489848-5fe526d1-7beb-4f2f-8577-ad774a138060.png">
